index.html

```diff
@@ -52,7 +52,113 @@ <h1>  I’m Lalla <br>
+                  <div class="content">
+                      <h3>Multi-Post Stories</h3>
+                      <ul>
+                          <li>Canopy</li>
+                          <li>Back End DEv</li>
+                          <li>2015</li>
+                      </ul>
+                      <p class="Descriptions">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                      <ul>
+                          <li><a href="#" class="programming">html</a></li>
+                          <li><a href="#" class="programming">css</a></li>
+                          <li><a href="#" class="programming">javascript</a></li>
+                      </ul>
+                      <button type="button">See Project</button>
+                  </div>
+              </div>
+              <div class="job">
+                <img src="image/Snapshoot-Portfolio-3.jpg" alt="job-3 image">
+                <div class="content">
+                    <h3>Tonic</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="Descriptions">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#" class="programming">html</a></li>
+                        <li><a href="#" class="programming">css</a></li>
+                        <li><a href="#" class="programming">javascript</a></li>
+                    </ul>
+
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job">
+                <img src="image/SnapshootPortfolio5.jpg" id="ICO">
+                <div class="content">
+                    <h3> Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="Descriptions">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#" class="programming">html</a></li>
+                        <li><a href="#" class="programming" >css</a></li>
+                        <li><a href="#" class="programming">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <section class="section-1">
+              <div class="sct-1">
+                  <h2>About  Myself</h2>
+                  <p class ="Descriptions"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] You "about me" section, still needs to be worked on to match the Figma design 🔽 please address the issues and fix them.

## Your Design
![2023-03-25_220556](https://user-images.githubusercontent.com/104319462/227744754-1f83035b-5c4c-4741-86b7-1ef36b32b41c.png)
## Figma Design
![2023-03-25_221013](https://user-images.githubusercontent.com/104319462/227744743-7d5b2450-17ed-4d0a-a9c6-a8f68f03bd9a.png)